manifests/init.pp

```diff
@@ -105,6 +105,16 @@
     notify  => Service[$freeradius::fr_service],
   }
 
+  # Add systemd unit to override default file on RHEL8 systems.
+  if ($facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '8') {
+    systemd::dropin_file { 'freeradius remove bootstrap':
+      ensure   => present,
+      filename => 'remove_bootstrap.conf',
+      unit     => 'radiusd.service', #@todo programmatically determine the service name
```

Comment:
Let's remove this todo / create an issue in its place?

Reply:
Or maybe we don't need to as this is only on el8, so it's always going to be radius.service..